client/app/components/context_menu/context-menu.service.js

```diff
@@ -17,79 +18,49 @@
             newFolder: {name: 'create-folder', title: 'New folder'},
             share: {name: 'share-folder', title: 'Share...'}};
 
-        var menu = {};
+        return {
+            getActions: getActions,
+            getTopPosition: getTopPosition,
```

Comment:
It actually was in the service. I just devided it into two methods: getTopPosition and getLeftPosition.